keps/sig-api-machinery/2885-server-side-unknown-field-validation/README.md

```diff
@@ -696,20 +692,22 @@ enhancement:
   cluster required to make on upgrade, in order to maintain previous behavior?
 - What changes (in invocations, configurations, API use, etc.) is an existing
   cluster required to make on upgrade, in order to make use of the enhancement?
+  -->
 
 ### Version Skew Strategy
 
-If applicable, how will the component handle version skew with other
-components? What are the guarantees? Make sure this is in the test plan.
+Following the graduation of server-side field validation to GA, there will be a
+period of time when a newer client (which expects server-side field validation locked in GA) will send
+requests to an older server that could have server-side field validation
+disabled. Until version skew makes it incompatible for a client to hit a server
+with field validation disabled, kubectl will need to continue to have
+client-side validation available. 
```

Comment:
The implications in this sentence seem incorrect.

> Until version skew makes it incompatible for a client to hit a server
with field validation disabled, kubectl will need to continue to have
client-side validation available.

1. Until version skew makes it incompatible for a client to hit a server with field validation disabled, kubectl MUST continue to check if the feature is enabled server-side.
2. As long as the feature can be disabled server-side, kubectl MUST (SHOULD?) continue to have client-side validation
3. When the feature can't be disabled server-side, kubectl SHOULD (MAY?) remove client-side validation